service/controller/cluster.go

```diff
@@ -59,6 +59,11 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 
 	var operatorkitController *controller.Controller
 	{
+		selector, err := labels.Parse(label.Organization)
```

Comment:
Select all Cluster CRs containing a non-empty `giantswarm.io/organization` label.

Reply:
Is there a possibility for a non CAPI cluster to having this? I think so. I am afraid that we might have competing operators then, but I am not 100% certain.

Reply:
I see what you mean. In the case of AWS with legacy clusters, `cluster-operator` should be the one installing apps. Having a tough time getting the right mix of labels...